doc/source/getting-involved.rst

```diff
@@ -37,6 +37,15 @@ Setting up your development environment
 
 To edit the Ray source code, you'll want to checkout the repository and also build Ray from source. Follow :ref:`these instructions for building <building-ray>` a local copy of Ray to easily make changes.
 
+When editing C++ code, if you're not very familiar with Google style, you may find it useful to repeatedly run clang-format as you go. Note however that ``clang-format`` will only pick up the ``.clang-format`` file if you run from the top-level directory with ``-style=file``.
```

Comment:
Actually the recommended solution is to use `ci/travis/format.sh`, which uses clang-format under the hood. 

Also, can you write the recommended clang-format version? I believe it is 7.0.0